helpers/search_processing_kendra_quickstart.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+set -o errexit
+set -o errtrace
+set -o pipefail
+set -o nounset
+
+# Some useful constants
+readonly DOCKER_IMAGE_TAG="opensearch-with-ranking-plugin"
+readonly OPENSEARCH_VERSION="2.4.0"
+
+function print_help() {
+  echo "Usage: $0 -p <execution_plan_id> [-r <region>] [-e <kendra_ranking_endpoint>]"
+  echo "  -p | --execution-plan-id          The ID returned from Kendra Intelligent Ranking service"
+  echo "                                    from the call to CreateRescoreExecutionPlan"
+  echo "  -r | --region                     The AWS region for the Kendra Intelligent Ranking"
+  echo "                                    service endpoint. If not specified, will read from the"
+  echo "                                    AWS CLI for the default profile."
+  echo "  -e | --kendra-ranking-endpoint    The URL for the Kendra Intelligent Ranking service"
+  echo "                                    endpoint. If not specified, a value based on the"
+  echo "                                    region will be used."
+  echo
+  echo "  NOTE: The script will attempt to read AWS credentials (access/secret key)"
+  echo "  from environment variables, and then from the default AWS profile, in order to"
```

Comment:
Can you add support for `--profile` rather than assume use of default profile only?

Reply:
Will do!

Reply:
Will do!